### ISSUE TITLE
Restyle/reformat pages with content editor in mind

### ISSUE THREAD

**Issue by gemfarmer** [18F/doi-extractives-data#1371]
The goal of the [Markdownify PR](https://github.com/18F/doi-extractives-data/pull/1368) was create pages that were predominantly markdown for ease-of-edit by future maintainers of this site. The premise of this move was that HTML is unfriendly to people who don't normally use it, and all of the extra brackets and unfamiliar notation can get in the way of actually doing the work – updating the content. 

As I began this conversion, some stuff was easy. Repeated layout containers can be obfuscated easily, some stuff is less straightforward. Our content pages are very similar to one another, but they all have their quirks. These quirks require page-specific styles that make using pure markdown challenging to impossible. @shawnbot made [a comment](https://github.com/18F/doi-extractives-data/pull/1368#issuecomment-209688581) linking to [this talk](https://vimeo.com/101718785) about "Effortless Style", an approach that focuses on attaching styles primarily to standard HTML tags and generating a custom feel by using clever css selectors that derive interesting layouts in a patterned way. Given our desire to create a comfortable environment for future content editors via markdown, I think *certain* EITI pages are perfect candidates for a bit of Effortless Style magic. 

The reason why this is a new issue, is that this isn't going to be as straightforward as just "converting" HTML to markdown. For the content pages, this will mean *slightly* redesigning the pages such that they can be styled algorithmically. 

Take for example a page that has five `<h2>` tags, some of which are styled as `h3`s like so: `<h2 class="h3">`. It will be a challenge to discern a pattern across pages for when to style an `h2` like an `h2` and when to style it like an `h3`. To implement the Effortless Style approach, we will need to either always order our headers properly, or create a predictable pattern for when they are styled differently. This is where @ericronne comes in. In order for this conversion to happen, I think Eric and I are going to need to chat big picture on how we style our content pages going forward.

**Comment by ericronne** [comment]
yes let's get something on the books asap @gemfarmer!
next week is very tight, but i'll find an hour for us to collaborate and get a plan in motion

**Comment by gemfarmer** [comment]
Great, @ericronne if you are busy, no rush. Find an hour and add it to my cal!

**Comment by gemfarmer** [comment]
From meeting with @ericronne:
We discussed general changes that would need to be made to make way for a markdown-dominated site. We decided that I would start making some changes to sets of similar pages to see if we can meet our needs stylistically without being tied to classes. 

Things to focus on as I am markdownifying pages:
- lists: disk, with equal spacing at the top of the list and between list items
- headers: 
  - move in logical order (for accessibility) and without custom `<h2 class='h3">` styling
  - pages should start with h1s
- larger text at head of page not important. Can be nixed
- focus on adding appropriate spacing above headers as a way to add appropriate spacing to paragraph sections
